amethyst_input/src/bindings.rs

```diff
@@ -63,19 +65,19 @@ where
         self.axes.keys().cloned().collect::<Vec<AX>>()
     }
 
-    /// Add a button to an action.
+    /// Add a button or button combination to an action.
     ///
-    /// This will insert a new binding between this action and the button.
-    pub fn insert_action_binding<A>(&mut self, id: A, binding: Button)
+    /// This will insert a new binding between this action and the button(s).
+    pub fn insert_action_binding<A>(&mut self, id: A, binding: SmallVec<[Button; 2]>)
```

Comment:
I'm not super sure I want to expose SmallVec here as part of our API, it should just be an implementation detail.  I noticed at one point this was implemented with iterators, why did we change that?

Reply:
I don't think `SmallVec` should be used here - http://troubles.md/posts/improving-smallvec/

Reply:
I'm not sure when or where it got lost, but i restored that too :)